tools/summarize-topic

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+import urllib.parse
+
+import zulip
+from litellm import completion
+
+tools_dir = os.path.dirname(os.path.abspath(__file__))
+root_dir = os.path.abspath(os.path.join(tools_dir, ".."))
+tools_test_dir = os.path.join(tools_dir, "tests")
+
+sys.path.insert(0, root_dir)
+
+# check for the venv
+from tools.lib import sanity_check
+
+sanity_check.check_venv(__file__)
+
+os.environ["HUGGINGFACE_API_KEY"] = "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"
```

Comment:
This is a read only API key of my account.

Reply:
Please never put an API key into a pull request. Just make it an option, and have the instructions to be `env HUGGINGFACE_API_KEY="{key}" ./command` or whatever if you need to pass it via the environment anyway.

Reply:
Okay, removed the API key.

I think it is better to use `os.environ["HUGGINGFACE_API_KEY"]` here since we will be playing with different models and it is easier to switch API keys for models this way.

Reply:
hmm, just wanted to point out that the API key is still visible in the outdated code comment preview here.

Reply:
Thanks for the ping. It is no longer a valid API key.